locales/en.yml

```
@@ -0,0 +1,225 @@
+server_details: "Server Details"
+server_name: "Server Name"
+server_url: "Server URL"
+choose_service: "Choose Service"
+cancel: "Cancel"
+login: "Login"
+logging_in: "Logging in..."
+login_to_service: "Login to %{service}"
+username: "Username"
+password: "Password"
+player_settings: "Player Settings"
+discord_presence: "Discord Presence"
```

YAMLlint (1.38.0)

[error] duplication of key "discord_presence" in mapping

(key-duplicates)